aopy/analysis.py

```diff
@@ -1398,6 +1400,43 @@ def match_selectivity_accLLR(test_data_altcond, train_data_altcond, train_data_n
 
         return noisy_test_data
 
+def classify_by_lda(X_train_lda, y_class_train, 
+                                 n_splits=5,
+                                 n_repeats=3, 
+                                 random_state=1):
+    """
+    train an linear discriminant model on the training data (X_train_lda) and their labels (y_class_train) with data spliting and
+    k-fold validation. 
```

Comment:
Can you add a bit more description of how it might be used? Something like 
```suggestion
    Trains a linear discriminant model on the training data (X_train_lda) and their labels (y_class_train) with data spliting and
    k-fold validation. Returns accuracy and variance based on how well the model is able to predict the left-out data.
```